pkg/util/fsnotification/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package fsnotification provides helper functions that wrapper the fsnotify filesystem
```

Comment:
it's just you.  it's a plural alignment thing:
one item wraps another
two items wrapper another